autofs-5.1.7 - remove unused function master_submount_list_empty()

From: Ian Kent <raven@themaw.net>

This function is not used anywhere now, remove it.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG        |    1 +
 daemon/master.c  |   12 ------------
 include/master.h |    1 -
 3 files changed, 1 insertion(+), 13 deletions(-)

--- autofs-5.1.4.orig/CHANGELOG
+++ autofs-5.1.4/CHANGELOG
@@ -45,6 +45,7 @@
 - remove redundant local var from sun_mount().
 - use mount_fullpath() in one spot in parse_mount().
 - pass root length to mount_fullpath().
+- remove unused function master_submount_list_empty().
 
 xx/xx/2018 autofs-5.1.5
 - fix flag file permission.
--- autofs-5.1.4.orig/daemon/master.c
+++ autofs-5.1.4/daemon/master.c
@@ -1119,18 +1119,6 @@ int master_read_master(struct master *ma
 	return 1;
 }
 
-int master_submount_list_empty(struct autofs_point *ap)
-{
-	int res = 0;
-
-	mounts_mutex_lock(ap);
-	if (list_empty(&ap->submounts))
-		res = 1;
-	mounts_mutex_unlock(ap);
-
-	return res;
-}
-
 int master_notify_submount(struct autofs_point *ap, const char *path, enum states state)
 {
 	struct mnt_list *this, *sbmnt;
--- autofs-5.1.4.orig/include/master.h
+++ autofs-5.1.4/include/master.h
@@ -116,7 +116,6 @@ void master_free_mapent_sources(struct m
 void master_free_mapent(struct master_mapent *);
 struct master *master_new(const char *, unsigned int, unsigned int);
 int master_read_master(struct master *, time_t);
-int master_submount_list_empty(struct autofs_point *ap);
 int master_notify_submount(struct autofs_point *, const char *path, enum states);
 void master_notify_state_change(struct master *, int);
 int master_mount_mounts(struct master *, time_t);
